lib/common/GenericTagTypes.ts

```diff
@@ -106,7 +106,16 @@ export type GenericTagId =
   | 'replaygain_track_minmax'
   | 'replaygain_album_minmax'
   | 'replaygain_undo'
-  | 'description';
+  | 'description'
+  | 'category'
+  | 'hdvideo'
+  | 'keywords'
+  | 'movement'
+  | 'movementnumber'
+  | 'movementnumbertotal'
+  | 'podcastid'
+  | 'showmovement'
+  | 'stik';
```

Comment:
Please also update corresponding the API documentation: [doc-gen/common.json](https://github.com/Borewit/music-metadata/blob/master/doc-gen/common.json).

I know _common_ properties have an inconsistent naming style (one of the reasons is backward compatibility), but please use apply camel case for new ones.